sdk/azcore/internal/shared/constants.go

```diff
@@ -31,6 +31,8 @@ const (
 
 const BearerTokenPrefix = "Bearer "
 
+const TracingNamespaceAttrName = "az.namespace"
```

Comment:
In anticipation of the unbranding, we may want to stop using "az" whenever we can